packages/story-editor/src/components/floatingMenu/menus/mixed.js

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { memo } from '@googleforcreators/react';
+
+/**
+ * Internal dependencies
+ */
+import { Dismiss, Separator } from '../elements';
+
+const FloatingTextMenu = memo(function FloatingTextMenu() {
```

Comment:
Should be something else than `MultiSelectionMenu` or something similar perhaps?

Reply:
Oh yes, forgot to rename this one. It's temporary anyway. Fixed in 391bb33717.